jina/serve/streamer.py

```diff
@@ -18,6 +18,25 @@
     from prometheus_client import CollectorRegistry
 
 
+def _create_topology_graph(
+        graph_description,
+    graph_conditions,
+    deployments_metadata,
+    deployments_no_reduce,
+    timeout_send,
+    retries,
+):
+    # check if it should be in K8s, maybe ConnectionPoolFactory to be created
+    return TopologyGraph(
+        graph_representation=graph_description,
+        graph_conditions=graph_conditions,
+        deployments_metadata=deployments_metadata,
+        deployments_no_reduce=deployments_no_reduce,
+        timeout_send=timeout_send,
+        retries=retries,
+    )
+
+
```

Comment:
I don't understand the purpose of this, it seems like it is the same as just calling the constructor directly?